docker-compose.yml

```diff
@@ -7,11 +7,12 @@ services:
       - ~/polkadot-data:/data
     # ports:
     #   - "9944:9944"
-    command: 'polkadot --chain polkadot --unsafe-ws-external --rpc-cors=all'
+    command: '--chain polkadot --unsafe-ws-external --rpc-cors=all'
```

Comment:
Did this line work at all previously (or did older images not default to running `polkadot`)?

Reply:
I did not check but it is **very** likely that I initially used my `chevdor/polkadot` image here. The difference with the parity one being that the parity image uses `polkadot` as entrypoint and my image does not. I guess someone updated from `chevdor` to `parity` without changing the call.

So the only 2 options are:
- use `chevdor/polkadot` **and** start  the command with `polkadot`
- use `parity/polkadot` and omit the `polkadot` binary in the command

My image is not built from CI so I do NOT recommend any longer using it (expect for people who want to build their own image themself).